cylc/flow/workflow_files.py

```diff
@@ -640,29 +648,7 @@ def load_contact_file(reg: str) -> Dict[str, str]:
     return data
 
 
-async def load_contact_file_async(reg, run_dir=None):
-    if not run_dir:
-        path = Path(get_contact_file_path(reg))
-    else:
-        path = Path(
-            run_dir,
-            WorkflowFiles.Service.DIRNAME,
-            WorkflowFiles.Service.CONTACT
-        )
-    try:
-        async with aiofiles.open(path, mode='r') as cont:
-            data = {}
-            async for line in cont:
-                key, value = [item.strip() for item in line.split("=", 1)]
-                # BACK COMPAT: contact pre "suite" to "workflow" conversion.
-                # from:
-                #     Cylc 8
-                # remove at:
-                #     Cylc 8.x
-                data[key.replace('SUITE', 'WORKFLOW')] = value
```

Comment:
Will removing this catch anyone out?

Reply:
It was a duplicate of what was already in `load_contact_file()` so not actually removed

Reply:
I've replaced this with the sync `load_contact_file` interface run inside a thread via `make_async`.